docs/src/components/App.js

```diff
@@ -14,15 +15,20 @@ const NavigationWithRouter = withRouter(Navigation);
 export default function App(props: Props) {
   const { children } = props;
   return (
-    <Box mdDisplay="flex" direction="row" minHeight="100vh">
-      <Column span={12} mdSpan={2}>
-        <Box padding={4}>
+    <Box minHeight="100vh">
+      <Header />
+
+      <Box smDisplay="flex" direction="row">
+        <Column span={12} smSpan={2}>
           <NavigationWithRouter />
-        </Box>
-      </Column>
-      <Column span={12} mdSpan={10}>
-        <Box padding={4}>{children}</Box>
-      </Column>
+        </Column>
+        <Divider />
```

Comment:
@chrislloyd nice trick - actually didn't know we could create a vertical divider like this.

Reply:
Neither did I until I tried it! I started off adding a vertical option to divider but then I realized that I hadn't actually rebuilt the library. Works well in the mobile case too.